test/tests/breadcrumb_index.js

```diff
@@ -2,7 +2,7 @@ const { ariaTest } = require('..');
 const { By } = require('selenium-webdriver');
 const assertAriaLabelExists = require('../util/assertAriaLabelExists');
 
-const exampleFile = 'breadcrumb/index.html';
+const exampleFile = 'content/patterns/breadcrumb/examples/breadcrumb.html';
```

Comment:
For consistency, should we also change the name of the test js file? "test/tests/breadcrumb_index.js" might lead someone to think this is for a different example, maybe?

Reply:
Great catch, I didn't think of that. Updated.